applications/nrf_desktop/configuration/common/hid_report_consumer_ctrl.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef _HID_REPORT_CONSUMER_CTRL_H_
+#define _HID_REPORT_CONSUMER_CTRL_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define REPORT_SIZE_CONSUMER_CTRL	1 /* bytes */
```

Comment:
2 bytes, update report mask (the same as for system control)

Reply:
Fixed